var/spack/repos/builtin/packages/vtkm/package.py

```diff
@@ -51,7 +52,11 @@ def cmake_args(self):
         with working_dir('spack-build', create=True):
             options = ["../",
                        "-DVTKm_ENABLE_TESTING:BOOL=OFF"]
-
+            # shared vs static libs
+            if "+shared" in spec:
+                options.append('-DBUILD_SHARED_LIBS=ON')
+            else:
+                options.append('-DBUILD_SHARED_LIBS=OFF')
```

Comment:
I dunno if you find it clearer, but FYI, this works (b/c CMake is case-insensitive and understands `True` and `False`):

```python
options.append('-DBUILD_SHARED_LIBS=%s' % ('+shared' in spec))
```